Dockerfile.debug

```diff
@@ -1,24 +1,28 @@
+# hadolint ignore=DL3007
 FROM golang:latest as build
 WORKDIR /go/src/github.com/pomerium/pomerium
 
+# hadolint ignore=DL3008
 RUN apt-get update \
-    && apt-get -y install zip
+    && apt-get -y --no-install-recommends install zip
 
 # cache depedency downloads
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
 
 # build
-RUN make build-deps
-RUN make build-debug NAME=pomerium
-RUN make build-debug NAME=pomerium-cli
-RUN touch /config.yaml
-RUN go get github.com/go-delve/delve/cmd/dlv
+RUN make build-deps && \
```

Comment:
See other comment re: build stage

Reply:
reverted in 3bb958f92366163fae3c64bf45149fa892cde0ab